### DISCUSSION TITLE
priority and snap in gridview panels

### DISCUSSION THREAD

**Discussion by husayt** [mathuo/dockview#579] [Q&A]
Hi, i see in source code there are 'priority' and 'snap' fields defined in GridView panels. But they are not used and make no effect.

Is it intentional? 'priority' at least would be very useful for GridView.

**Comment by uptonking** [comment]
as shown in this demo, `snap` can be used to make sidebar width shrink to 0.

https://github.com/mathuo/dockview/tree/master/packages/docs/sandboxes/editor-gridview
